week-2/I-array-properties/exercise.js

```diff
@@ -4,9 +4,12 @@
   Complete the function to test if an array is empty (has no values in it)
 
 */
-
 function isEmpty(arr) {
-  return; // complete this statement
+  if (arr == 0) {
```

Comment:
When making comparisons, we usually try to avoid loose comparisons with `==`. JavaScript will try to compare an `Array` to an `Number`. In this case, it does work, but you'll find in a lot of cases comparing across types leads to unexpected results.

Instead, we like to compare an `Number` with an `Number`. Here it should be `arr.length === 0`, which will give us a `Number` (`arr.length`) to compare with `0`.